### PR TITLE
fix: Add french translations

### PR DESCRIPTION
**Pull request by louisgreiner**
# Description

Fill the french translation table

Do not hesitate to use `npm run start:standalone` for quick review, if the whole project is long to instanciate

# Issues

Not so sure about business oriented wording

# Open discussion

Choices have to be made about a few business words, @aiAdrian:
- "Trainrun" (EN) -> "Ligne de train" (FR) or "Trajet de train" (FR) ou "Course de train" (FR) ? As NGE uses also "Trainrun" (EN), even for the German version. Also, "Zugfahrt" (DE) doesn't exactly mean "Trainrun" (EN) right ?
- "Netzgrafik" (DE) -> "Réticulaire" (FR) or "Graphique de réseau" (FR) ? (At SNCF, we mostly use "Réticulaire" (FR), https://www.sncf-reseau.com/medias-publics/2023-11/R%C3%A9ticulaire%20PER%20Marseille-Vintimille%20SA%202025%20-%2030%20Novembre%202023.pdf)
- "Label" (DE/EN) -> "Label" (FR) or "Etiquette" (FR) ?

# Checklist

* [x] This PR contains a description of the changes I'm making
* [x] I've read the [Contribution Guidelines](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/blob/main/CONTRIBUTING.md)
* [x] CI is currently green and this is ready for review

### CODE REVIEW

**Comment by louisgreiner** [comment]
I cannot tag you, but if you could help it would be very nice, since you've got more experience in the matter i guess :) @SergeCroise

**Comment by SergeCroise** [comment]
> I cannot tag you, but if you could help it would be very nice, since you've got more experience in the matter i guess :) @SergeCroise

ok!

File: src/assets/i18n/fr.json [hunk 1]
```diff
@@ -2,645 +2,644 @@
...
-          "search-variants": "Search variants",
-          "show-archive": "Show archive",
-          "project-description": "Project description",
-          "edit-project": "Edit project",
+          "archived": "(archivée)",
+          "variants": "Variantes",
+          "search-variants": "Rechercher des variantes",
+          "show-archive": "Afficher les archives",
+          "project-description": "Description du projet",
+          "edit-project": "Editet le projet",
```
Comment (SergeCroise) [inline src/assets/i18n/fr.json:603]:
```suggestion
          "edit-project": "Editer le projet",
```

File: src/assets/i18n/fr.json [hunk 2]
```diff
@@ -2,645 +2,644 @@
...
+        "delete-all-visible-trainruns-tooltip": "Supprimer toutes les lignes de trains visibles",
+        "delete-all-visible-trainruns": "Supprimer toutes les lignes de trains visibles",
+        "delete-all-visible-notes-tooltip": "Supprimer toutes les notes visibles",
+        "delete-all-visible-notes": "Supprimer toutes les notes visibles",
+        "delete-all-visible-elements-tooltip": "Supprimer tous les éléments visibles, tels que les noeuds, les lignes de train et les notes",
+        "delete-all-visible-elements": "Supprimer tous les éléments visibles",
+        "merge-netzgrafik-title": "Netzgrafik - Fusionner",
+        "add-netzgrafik-as-copy-tooltip": "Ajouter un netzgrafik en tant que copie (lignes de train, noeuds, notes)",
+        "add-netzgrafik-as-copy": "Ajouter un netzgrafik en tant que copie",
+        "merge-netzgrafik-tooltip": "Fusionner les réticulaires (lignes de train, noeuds, notes)",
```
Comment (SergeCroise) [inline src/assets/i18n/fr.json:292]: réticulaires ?

**Review by SergeCroise — CHANGES_REQUESTED** [review]
fr.json:
pour la traduction de trainrun (Zugfahrt), j'hésite entre
**ligne de train**
et
**trajet de train**
qu'en pensent les autres francais  qui participent au projet osrd ?
de.json:
in de.json, I suggest to replace
Variante-ID 
with
Varianten-ID
and
Unendlichkeitsschleife
with
Endlosschleife

File: src/assets/i18n/fr.json [hunk 3]
```diff
@@ -2,645 +2,644 @@
...
-        "hide-advanced-display-functions": "Hide advanced display functions",
-        "mainTracks": "Main tracks",
-        "headway-band": "Headway band",
-        "time": "Time",
-        "name": "Name"
+        "reset": "Réinitialiser",
+        "close-all-path-nodes": "Fermer tous les noeuds du chemin",
+        "open-all-path-nodes": "Ouvrir tous les noeuds du chemin",
+        "show-advanced-display-functions": "Afficher les fonctionnalités d'affichage avancées",
+        "show-or-hide-time-for-non-selected-trainruns": "Afficher/masquer le temps des lignes de train non sélectionnés",
```
[the remark this answers is not in the document]
Comment (louisgreiner) [reply src/assets/i18n/fr.json:128]: Thanks, I'll put that in open discussion in the PR's body

File: src/assets/i18n/fr.json [hunk 4]
```diff
@@ -2,645 +2,644 @@
...
-        "display-arrival-departure-times": "Display departure and arrival times",
-        "display-arrival-departure-times-for-connections": "Display departure and arrival times for connections",
-        "display-travel-times": "Display travel times",
-        "display-trainrun-name": "Display trainrun name",
-        "connections": "Connections",
-        "display-connections": "Display connections",
-        "nodes": "Nodes",
-        "display-nodes-without-visible-trainruns": "Display nodes without (visible) trainruns",
-        "display-nodes-with-only-stopping-trainruns": "Display nodes with only stopping trainruns",
+        "filter": "Filtrer",
```
[the remark this answers is not in the document]
Comment (louisgreiner) [reply src/assets/i18n/fr.json:309]: as in the german file, the word is "Filter", I guess it should refer to the noun

File: src/assets/i18n/fr.json (hunk 2 above)
Comment (louisgreiner) [reply src/assets/i18n/fr.json:292]: "Réticulaire" means network graphic in french, as "Graphique de réseau" has a quite large range of meaning. It is actually the word used by the business people, at least at SNCF Réseau:
https://www.sncf-reseau.com/medias-publics/2023-11/R%C3%A9ticulaire%20PER%20Marseille-Vintimille%20SA%202025%20-%2030%20Novembre%202023.pdf

I'll put that in the open-discussion list

**Comment by aiAdrian** [comment]
> fr.json: pour la traduction de trainrun (Zugfahrt), j'hésite entre ligne de train, course de train (https://osrd.fr/fr/docs/explanation/netzgrafik-editor/) et trajet de train qu'en pensent les autres francais qui participent au projet osrd ?
> 
> de.json: in de.json, I suggest to replace Variante-ID with Varianten-ID and Unendlichkeitsschleife with Endlosschleife what is your opinion @aiAdrian ? see also
> 
> ```
>         "add-netzgrafik-as-copy-tooltip": "Netzgrafik als Kopie hinzufügen (Zugfahrtenn, Knoten, Kommentare)",
>         "add-netzgrafik-as-copy": "Netzgrafik als Kopie hinzufügen",
>         "merge-netzgrafik-tooltip": "Netzgrafik durch Zusammenführen mergen (Zugfahrtenn, Knoten, Kommentare)",
>         "merge-netzgrafik": "Netzgrafik durch Zusammenführen mergen",
> ```

@SergeCroise

*Endlosschleife* -> is correct -> thanks.
*Varianten-ID* -> is correct -> thanks.


```
        "add-netzgrafik-as-copy-tooltip": "Netzgrafik als Kopie einfügen (Zugfahrten, Knoten, Kommentare)",
        "add-netzgrafik-as-copy": "Netzgrafik als Kopie einfügen",
        "merge-netzgrafik-tooltip": "Netzgrafik zusammenführen (Zugfahrten, Knoten, Kommentare)",
        "merge-netzgrafik": "Netzgrafik durch zusammenführen",
```

**Review by aiAdrian — APPROVED** [review]
Thanks - looks good for me

**Pull request merged**